common/static/karma_common.conf.js

```diff
@@ -41,7 +41,9 @@ var options = {
 
         {pattern: 'js/vendor/jasmine-imagediff.js', included: true},
         {pattern: 'common/js/spec_helpers/jasmine-extensions.js', included: true},
-        {pattern: 'common/js/spec_helpers/jasmine-waituntil.js', included: true}
+        {pattern: 'common/js/spec_helpers/jasmine-waituntil.js', included: true},
+        {pattern: 'common/js/spec_helpers/discussion_spec_helper.js', included: true},
+        {pattern: 'common/js/spec/discussion/view/discussion_view_spec_helper.js', included: true}
```

Comment:
We'll want to clean this up eventually and figure out a consistent place for spec helpers to go but for now I say merge ahead and get this in before the release.